2-mandatory/2-financial-times.js

```diff
@@ -5,16 +5,33 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    const shortArticleTitles = [];
+    for (let articleTitle of allArticleTitles) {
+        if (articleTitle.length <= 65) {
+            shortArticleTitles.push(articleTitle);
+        }
+    }
+    return shortArticleTitles;
 }
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    
+    let fewestword = allArticleTitles[0].split(' ').length;
+    let result;
+    for (let articleTitle of allArticleTitles) {
+       let articleWords = articleTitle.split(' ').length;
+       if (articleWords < fewestword ) {
```

Comment:
`if (articleWords <= fewestword)`
to handle the case when the first element of the array is the fewest word.

Reply:
Thank you for pointing me to it.